src/universal/components/Button/Button.js

```diff
@@ -76,7 +77,7 @@ const Button = props => {
     buttonOptions.push(styles.isBlock);
   }
 
-  const buttonStyles = combineStyles.apply(null, buttonOptions);
+  buttonStyles = combineStyles.apply(null, buttonOptions);
```

Comment:
Seems fine to me. Basically I want to push styles into an array, the apply that array using the `combineStyles` method (instead of managing comma-separated params and multiple combineStyle calls).